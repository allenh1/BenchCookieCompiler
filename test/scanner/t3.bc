[ double; ++7; ------8; 4848484; 123.34; maybe int ? capture; ]
