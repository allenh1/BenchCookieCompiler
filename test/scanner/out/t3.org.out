{<OBRACKET>: [}
{<DOUBLE>: double}
{<SEMICOLON>: ;}
{<PLUS>: +}
{<PLUS>: +}
{<INTEGER>: 7}
{<SEMICOLON>: ;}
{<MINUS>: -}
{<MINUS>: -}
{<MINUS>: -}
{<MINUS>: -}
{<MINUS>: -}
{<MINUS>: -}
{<INTEGER>: 8}
{<SEMICOLON>: ;}
{<INTEGER>: 848494}
{<SEMICOLON>: ;}
{<FLOAT>: 123.34}
{<SEMICOLON>: ;}
{<MAYBE>: maybe}
{<INT>: int}
{<QUESTION>: ?}
{<CAPTURE>: capture}
{<SEMICOLON>: ;}
{<CBRACKET>: ]}
